google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/ConsistencyParams.java

```diff
@@ -0,0 +1,68 @@
+package com.google.cloud.bigtable.admin.v2.models;
+
+import com.google.api.core.InternalApi;
+import com.google.bigtable.admin.v2.CheckConsistencyRequest;
+import com.google.bigtable.admin.v2.TableName;
+import com.google.common.base.Preconditions;
+
+import javax.annotation.Nonnull;
+
+public class ConsistencyParams {
```

Comment:
for better or for worse, the current naming convention is to suffix with Request. So I think should be ConsitencyRequest